packages/jest-environment-jsdom/src/index.js

```diff
@@ -22,13 +22,14 @@ class JSDOMEnvironment {
   errorEventListener: ?Function;
   moduleMocker: ?ModuleMocker;
 
-  constructor(config: ProjectConfig): void {
+  constructor(config: ProjectConfig) {
     const jsdomInitialized = process.hrtime();
-    // lazy require
-    this.document = JSDom.jsdom('<!DOCTYPE html>', {
+
+    this.document = new JSDOM('<!DOCTYPE html>', {
+      runScripts: 'dangerously',
       url: config.testURL,
     });
-    const global = (this.global = this.document.defaultView);
+    const global = (this.global = this.document.window.document.defaultView);
```

Comment:
Wait, why do this?

Reply:
Oh, I see, `this.document` is no longer a document, it's a `JSDOM` instance. That's pretty confusing.

Reply:
Which part is confusing? Should `this.document` be renamed to `this.jsdomInstance`? I have to admit I floundered a bit here

Reply:
Yeah, you shouldn't name a JSDOM instance `document` (similar to how you shouldn't name it `birthdayCake`).

We use the variable name `dom` for short in the JSDOM docs.